.github/workflows/ci.yml

```diff
@@ -97,8 +97,6 @@ jobs:
     needs: [test]
     uses: OpenAstronomy/github-actions-workflows/.github/workflows/publish.yml@main
     with:
-      test_extras: 'dev'
-      test_command: 'pytest -p no:warnings --doctest-rst -m "not mpl_image_compare" --pyargs sunpy'
```

Comment:
If we remove these, the `sdist` won't be tested. However, if we keep these, cibuildwheel will use these over the values set in `pyproject.toml`.

I do like the idea of including the packaging test commands inside the package (for reproducible builds). If this is the goal we could add a custom entry into `pyproject.toml` and parse it ourselves in a separate job during the workflow run. (Or we could not prune `.github` in the manifest.)